administration/src/bp-modules/regions/data-privacy-policy/DataPrivacyOverview.tsx

```diff
@@ -68,14 +64,24 @@ const DataPrivacyOverview = ({ dataPrivacyPolicy, regionId }: RegionOverviewProp
   return (
     <>
       <Content>
-        <Label>{t('dataPrivacy')}</Label>
-        <TextArea
-          fill
-          onChange={e => setDataPrivacyText(e.target.value)}
-          value={dataPrivacyText}
-          large
-          rows={20}
+        <Typography mt={5} variant='h5' align='center'>
+          {t('dataPrivacy')}
+        </Typography>
+        <TextField
+          sx={{
+            mb: 0,
+          }}
+          slotProps={{
+            input: {
+              sx: { fontSize: theme.typography.body2.fontSize },
+            },
+          }}
```

Comment:
❓ I'm not sure, why we are using `body2` everywhere instead of the default `body`, but I think it would be possible to define this is in the `mui-modules/theme.ts` file.

Maybe we can later sort this out in the theme without going through every component later.